locale/panes/bulkOp/en.yaml

```diff
@@ -18,3 +18,4 @@ headers:
                 =1 {one person}
                 other {# persons}}
 submitButton: Perform operation
+tooLargeActionExport: You can at most export { maxActionsSupportedInExport } Actions.
```

Comment:
I think it would make sense to have this under `action.export` in the YAML structure, in which case it can also have a shorter key name, like `action.export.tooLarge`.